packages/app-accounts/src/Restore.tsx

```diff
@@ -142,6 +144,13 @@ class Restore extends React.PureComponent<Props, State> {
     try {
       const pair = keyring.restoreAccount(json, password);
 
+      onStatusChange({
+        action: 'restore',
+        success: !!(pair),
+        value: pair.address(),
+        message: 'Restored'
```

Comment:
All messages should be translatable, i.e. `t('status.restored', { defaultValue: 'Restored' })` (The same goes for all places where we have string in this PR, i.e. forget, edited, created, restored...)